distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -726,6 +729,85 @@ type Table
         result = Table.new new_columns
         on_problems.attach_problems_after result problem_builder.to_vector
 
+    ## Replaces the first, last, or all occurrences of term with new_text in each text row of selected columns.
+       If `term` is empty, the function returns the table unchanged.
+
+       This method follows the exact replacement semantics of the `Text.replace` method.
+
+       Arguments:
+       - columns: The columns of the table in which to perform text replacement.
+       - term: The term to find.
+       - new_text: The new text to replace occurrences of `term` with.
+         If `matcher` is a `Regex_Matcher`, `new_text` can include replacement
+         patterns (such as `$<n>`) for a marked group.
+       - mode: Specifies which instances of term the engine tries to find. When the
```

Comment:
very minor nitpick but wouldn't `occurrence` be a better matching word rather than `instance` in this context?

Reply:
Big chunk of this docs is copied from `Text.replace`, with only slight modifications to adapt it to table. But I agree that `occurence` works better here. I can backport those changes there as well.

Reply:
I think it may be worth to do so :) Thanks!